CHANGELOG.md

```diff
@@ -32,7 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Minor out-of-date code removed or updated
 - Change the sidebar link so it only works when clicked directly
   (and not when the empty space next to it is clicked)
-- Show revision size in popup even when there's no revision comment
+- Fetch username from the MediaWiki API, and fade-in all content in the revision popup.
```

Comment:
0.11.0 has already been released, and shouldn't have it's changelog changed. New changes should go above in 'Unreleased'.

Reply:
Haha, I thought I was under the Unreleased heading, and just one of the release notes were cancelled out by this one. Fixed!